build/root/.bazelrc

```diff
@@ -1,5 +1,8 @@
 startup --expand_configs_in_place
 
+# don't error on error on deprecated native http_archive rule
+build --incompatible_remove_native_http_archive=false
```

Comment:
I think this might be getting removed in bazel 0.23.0, but we're pinned to an old version of bazel on CI anyway.